Automattic/woocommerce-payments
# Multiple small fixes for i18n issues

Fixes #6464 

This is a continuation of https://github.com/Automattic/woocommerce-payments/pull/6385. The older PR was behind the main branch and it was a pain to rebase it with all the new changes. This is a fresh start with cherry-picks from the older branch.

#### Changes proposed in this Pull Request
- Making sure most phrases are ready to be translated by wrapping them in our i18n helpers.
- Making sure translator comments are included with each translation
- **NEW**: Adds a mention to check i18n string to the new PR template

#### Testing instructions





* Make sure all changes look good
* Run `npm run build`
* Open `languages/woocommerce-payments.pot` and make sure the new phrases are showing correctly. (It's easy to copy a phrase and searching for it)

-------------------


**Post merge**
